cetmix_tower_server/models/cx_tower_variable_value.py

```diff
@@ -90,6 +99,21 @@ class TowerVariableValue(models.Model):
         ),
     ]
 
+    @api.depends("variable_id", "option_id")
+    def _compute_value_char(self):
+        for rec in self:
+            if rec.variable_id.option_ids and rec.option_id:
+                rec.value_char = rec.option_id.name
+
+    @api.depends("option_id", "variable_id")
```

Comment:
Same as above: we need to deepend on fields that are being used in the computation.

Reply:
I will make a change